DqtNotifications/terraform/data.tf

```diff
@@ -0,0 +1,19 @@
+data "azurerm_resource_group" "rgsb" {
```

Comment:
Not sure what `rgsb` stands for, could we use something simpler, `resource_group` maybe